content/guides/api-overview.md

```diff
@@ -0,0 +1,75 @@
+# Smily Channel API Overview
+
+1. TOC
+{:toc}
+
+## What is Smily Channel API?
+
+The channel API helps channel partners to integrate with our API by providing “shortcuts” so that partners can pull off the required data they need without having to combine too many requests from our API.
+
+## Who is Smily Channel API for?
+
+The Smily Channel API is designed to be useful for a variety of platforms. It's particularly valuable for:
+
+  - **Online Travel Agencies (OTAs)**
+  - **Marketplaces**
+  - **Tour Operators**
+  - **Niche Channels**
+
+**Create Your Own Portal:** Additionally, the Smily offers the flexibility to build entire Marketplaces or Niche Channels according to your vision. Explore this capability further on our [Your Own Portal](https://www.smily.com/en/your-own-portal) page.
```

Comment:
I think we also need to mention that for any other type of integration better use our [API V3 docs](https://developers.bookingsync.com/) for those partners

Reply:
We already mentioned it here - https://a.cl.ly/04uvz0KW. so we can skip it.